<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>academicrecordwindow</class>
 <widget class="QWidget" name="academicrecordwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #1e1e2f;
    color: #e0e0e0;
    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
    font-size: 14px;
}

QGroupBox {
    border: 1px solid #3a3f4b;
    border-radius: 8px;
    margin-top: 20px;
    padding: 12px 16px;
    background-color: #2b2b3c;
    color: #e0e0e0;
}

QGroupBox:title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 6px;
    font-weight: bold;
    font-size: 15px;
    color: #ffffff;
}

QLabel {
    color: #d6d6d6;
    font-size: 14px;
    font-weight: 500;
    padding: 2px 4px;
}

QLabel#sectionTitle {
    font-size: 18px;
    font-weight: bold;
    color: #87cefa; /* light sky blue */
    padding: 4px 0;
}


QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 24px;
    font-size: 15px;
    font-weight: 600;
}

QPushButton:hover {
    background-color: #357ab8;
}

QPushButton:pressed {
    background-color: #2e5c9a;
}
</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>431</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Диплом</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Учет оценок</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnCourseworkStart">
      <property name="text">
       <string>Курсовая</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnLessonsStart">
      <property name="text">
       <string>Начать работу</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Зачеты</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnTestStart">
      <property name="text">
       <string>Поставить зачет</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Проект</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btnDiplomStart">
      <property name="text">
       <string>Диплом</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Курсовая</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnProjectStart">
      <property name="text">
       <string>Проект</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Прочитать перед работой</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
